apis/k8ssandra/v1alpha1/dseconfig_types.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+type DSEYaml struct {
+
+	// TODO
+	Foo string `json:"foo,omitempty"`
```

Comment:
Just a placeholder for now. Actual support for all dse.yaml options will come later.